static/js/controllers/contacts-content.js

```diff
@@ -282,7 +293,10 @@ var _ = require('underscore'),
 
                 selected.fields = selectedSchemaVisibleFields(selected);
 
-                sortChildPersons(children.persons, contactDoc);
+                if (!children.persons) {
+                  children.persons = [];
+                }
```

Comment:
Is there a reason this logic has been moved outside of `sortChildPersons`? I feel like that function should be able to guard against this kind of thing itself. That would reduce possible bugs in the future if this code gets moved around and this if block gets disconnected / moved away from the `sortChildPersons` all.